x-pack/plugins/cases/public/containers/use_delete_cases.tsx

```diff
@@ -31,6 +31,7 @@ export const useDeleteCases = () => {
       onSuccess: (_, { successToasterTitle }) => {
         queryClient.invalidateQueries(casesQueriesKeys.casesList());
         queryClient.invalidateQueries(casesQueriesKeys.tags());
+        queryClient.invalidateQueries(casesQueriesKeys.categories());
```

Comment:
Probably you left it for another PR but let's put the same to `useRefreshCases`, `useRefreshCaseViewPage`, and `useUpdateCases` on this PR.